### PR TITLE
feature(databend): enhance norec, tlp, and implement pqs

### PR DESCRIPTION
**Pull request by hanyisong**

### CODE REVIEW

**Comment by hanyisong** [comment]
cc@mrigger There is an unresolved bug in databend

**Comment by hanyisong** [comment]
exception: Communications link failure
It is because databend stack overflow when executing tlp
```
thread 'mysql-query-executor' has overflowed its stack
fatal runtime error: stack overflow
```

**Comment by mrigger** [comment]
Sorry for the delay in reviewing this PR. I was on the road and will have a look asap.

File: src/sqlancer/databend/DatabendErrors.java [hunk 1]
```diff
@@ -10,12 +10,18 @@ private DatabendErrors() {
     public static void addExpressionErrors(ExpectedErrors errors) {
         errors.add("Division by zero");
         errors.add("/ by zero");
+        errors.add("ORDER BY position");
+        errors.add("GROUP BY position");
+        errors.add("downcast column error"); // bug
```
Comment (mrigger) [inline src/sqlancer/databend/DatabendErrors.java:15]: Rather then ignoring the bugs here, could you please use the pattern described at https://github.com/sqlancer/sqlancer/blob/master/CONTRIBUTING.md#unfixed-bugs?

**Review by mrigger — COMMENTED** [review]
Thanks a lot for the PR! For ignoring bugs, could you see whether you can adopt the pattern described in the linked page of the inline comment?

**Review by mrigger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**